includes/ParserPowerSimple.php

```diff
@@ -562,4 +567,95 @@ public static function arraymaptemplateRender($parser, $frame, $args) {
 		return implode( $new_delimiter, $results_array );
 	}
 	
+	public static function argmapRender($parser, $frame, $args) {
+		if ( !isset($args[0]) ) return "";
+
+		// sort arguments, this is to disregard the position of named arguments 
+		$named_args = [];
+		$unnamed_args = [];
+		foreach ($args as $arg) {
+			$arg = $frame->expand($arg);
+			if (strpos($arg, '=') !== false) {
+				list($key, $value) = explode('=', $arg, 2);
+				$named_args[trim($key)] = trim($value);
+			} else {
+				$unnamed_args[] = $arg;
+			}
+		}
+
+		// set and clean parameters
+		$formatter = isset($unnamed_args[1]) ? trim($frame->expand($unnamed_args[0])) : '';
+		$glue = isset($unnamed_args[1]) ? trim($frame->expand($unnamed_args[1])) : '';
+		$must_contain = isset($unnamed_args[2]) ? trim($frame->expand($unnamed_args[2])) : '';
+		$must_contain = ( $must_contain == '' ) ? [] : explode( ',', $must_contain );
+		$only_show = isset($unnamed_args[3]) ? trim($frame->expand($unnamed_args[3])) : '';
+		$only_show = ( $only_show == '' ) ? [] : explode( ',', $only_show );
```

Comment:
Use strict comparisons - i.e. `===` instead of `==`. Loose comparisons usually result in bugs as data may end up getting converted and you can end up with a string `'5'` matching `$a > 0` in integer operations.